lntest/fee_service.go

```diff
@@ -81,8 +85,9 @@ func NewFeeService(t *testing.T) *FeeService {
 	mux.HandleFunc("/fee-estimates.json", f.handleRequest)
 
 	f.srv = &http.Server{
-		Addr:    listenAddr,
-		Handler: mux,
+		Addr:              listenAddr,
+		Handler:           mux,
+		ReadHeaderTimeout: httpReadHeaderTimeout,
```

Comment:
nit: could use `lnd.DefaultHTTPHeaderTimeout` here directly. Or do you run into the "line too long" linter then?

Reply:
Missed this one. Changed it now.